content/applications/inventory_and_mrp/inventory/product_management/product_tracking/serial_numbers.rst

```diff
@@ -290,4 +352,12 @@ just one product per serial number.
    number.
 
 .. seealso::
-   :doc:`../product_tracking`
+   :doc:`Reassign <reassign>`
+
+.. tip::
+   In addition to using the :guilabel:`Lots/Serial Numbers` dashboard, there are several other
+   reporting templates that display the :guilabel:`Lot/Serial Number` field or the ability to filter
+   by serial number. Go to :menuselection:`Inventory app --> Reporting` to access:
+   - :guilabel:`Locations` report
+   - :guilabel:`Moves History` report
+   - :guilabel:`Moves Analysis` report
```

Comment:
this did not render properly in the HTML

I would also include it in the **seealso** block above, not as a **tip**

```suggestion
   :doc:`Reassign <reassign>`

   In addition to using the :guilabel:`Lots/Serial Numbers` dashboard, there are several other
   reporting templates that display the :guilabel:`Lot/Serial Number` field or the ability to filter
   by serial number. Go to :menuselection:`Inventory app --> Reporting` to access:
   
   - :guilabel:`Locations` report
   - :guilabel:`Moves History` report
   - :guilabel:`Moves Analysis` report
```